app/controllers/verify_controller.rb

```diff
@@ -21,12 +21,16 @@ def activated
   end
 
   def fail
-    redirect_to verify_url unless idv_attempter.exceeded?
+    redirect_to verify_url unless ok_to_fail?
   end
 
   private
 
   def active_profile?
     current_user.active_profile.present?
   end
+
+  def ok_to_fail?
```

Comment:
there's a method called `fail` and it has specific `before_action`-type restrictions.